rpcs3/Emu/Cell/lv2/sys_time.cpp

```diff
@@ -207,7 +207,8 @@ error_code sys_time_get_current_time(vm::ptr<s64> sec, vm::ptr<s64> nsec)
 	// get time since Epoch in nanoseconds
 	const u64 time = s_time_aux_info.start_ftime * 100u + (diff * g_cfg.core.clocks_scale / 100u);
 
-	*sec  = time / 1000000000ull;
+	// scale to seconds, and add the console time offset (which might be negative)
+	*sec  = (time / 1000000000ull) + g_cfg.sys.console_time_offset;
```

Comment:
```suggestion
	*sec = (time / 1000000000ull) + g_cfg.sys.console_time_offset;
```

Reply:
I was wondering.... does this work if you end up getting a *sec that's negative or in some ridiculous area?

Reply:
I suspect it will eventually end badly.
There's not a huge risk of the negative situation here, since the QDateTimeEdit widget doesn't allow for an entry to be put in before 1752... I'm unsure what the PS3 would do if given such a time, and games may or may not handle it well (albeit it is a valid time according to PS3 SDK).

The future time is more likely to be a problem I think.  Since the PS3 does report time starting from a ridiculous base time 1/1/0000 00:00.  That means given a 64 bit second value, the system ends up having used lots to get us to 2020.
So... it might become an issue.  But one that is already present with the ability to adjust the computer system clock.